tools_docker/Debian_Uwsgi/entrypoint.sh

```diff
@@ -54,23 +101,26 @@ case "$DATABASE_TYPE" in
     fi
     ;;
 esac
-
-# Initialize database if needed
+loggood "checking of database success"
+# Initialize database if needed
 if [ "$INIT_DATABASE" = true ] ; then
+    log "Initialise Database"
     cd /tracim/backend/
-    tracimcli db init -c /etc/tracim/development.ini
+    tracimcli_as_user "db init"
+    loggood "Database initialisation success"
 else
+    log "Update database"
     cd /tracim/backend/
-    alembic -c /etc/tracim/development.ini upgrade head
+    alembic_as_user "upgrade head"
 fi
-
-chown www-data:www-data -R /var/tracim
+loggood "database ready"
 
 # activate apache modules
 a2enmod proxy proxy_http proxy_ajp rewrite deflate headers proxy_html dav_fs dav expires proxy_uwsgi
 
 # Activate or deactivate webdav
 if [ "$START_WEBDAV" = "1" ]; then
+    log "Activation of webdav"
```

Comment:
Maybe log can explain `Creation of webdav configuration file` or something like that.